README.md

```diff
@@ -35,6 +35,7 @@ ENV.string("APP_NAME", default: "local")
 ENV.symbol("BUSINESS_DOMAIN", default: :engineering, required: true)
 ENV.boolean("ENABLE_FEATURE_FOO", default: false)
 ENV.integer("MAX_THREAD_COUNT", default: 5)
+ENV.file_handle("FILE_PATH", default: "DEFAULT_PATH", required: true)
```

Comment:
This is an example usage, can we put something that looks path-like here? Maybe `/tmp/file.json` or something like that?